fdbclient/ReadYourWrites.actor.cpp

```diff
@@ -1823,6 +1823,25 @@ Future<Standalone<VectorRef<BlobGranuleChunkRef>>> ReadYourWritesTransaction::re
 	return waitOrError(tr.readBlobGranules(range, begin, readVersion, readVersionOut), resetPromise.getFuture());
 }
 
+Future<Standalone<VectorRef<BlobGranuleSummaryRef>>> ReadYourWritesTransaction::summarizeBlobGranules(
```

Comment:
I wondering why you did not need to introduce previously for simulation tests. Are they not testing RYW transactions?

Reply:
That's correct, they were just calling the NativeApi's summarizeBlobGranule call.

Reply:
Not related to this PR, but we should try to cover ReadYourWritesTransaction with simulation tests too.